_data/projects.yaml

```diff
@@ -51,8 +51,8 @@
 
 - name: Spryker
   url: https://academy.spryker.com/what_s_new/releases/release_notes_august_1_2017.html#announcements
-  when: ~ 2018
-  done: false
+  when: Jan 2018
+  done: true
```

Comment:
Could you add link to this release? I could now find it

Reply:
The info in the linked academy doc is still as valid as it was August 2018 ( https://academy.spryker.com/what_s_new/releases/release_notes_august_1_2017.html#announcements ):
1.11.2017 was the first day everything become 7.1+
We just didnt tag releases for all existing 200 modules, but only pushed those to master (as releasing this for fun makes no sense to anyone, only for the next actual release of functionality).

Reply:
Now I understand what you mean. Don't try to look into the master itself, we do a split and release process ( https://academy.spryker.com/understanding_spryker/atomic_releases.html ), so we have new modules inside a subfolder, git subtree split and released standalone.

Reply:
If `done` = `true` and it's not clear, since which version that takes places.
The link should go to stable tag release with 7.1 as minimum.

Reply:
Yes, and this is all done.
As just explained, the subtree repo of https://github.com/spryker/spryker/tree/master/Bundles/ProductOption
is for example released as https://github.com/spryker/product-option/releases

So each folder has its own https://github.com/spryker/spryker/blob/master/Bundles/ProductOption/composer.json#L6 and therefore upon release it all released as 7.1+ just as communicated in August last year, doing so since November 2017.

Reply:
I see only "https://academy.spryker.com/what_s_new/releases/release_notes_august_1_2017.html#announcements"
No link to release

Reply:
See https://gophp71.org/, Symfony is the best example